### PR TITLE
Pr add am32 tune edit

### PR DESCRIPTION
**Pull request by Huibean**
add am32 tune edit gui
image

### CODE REVIEW

**Comment by Huibean** [comment]
have trouble handle empty tune from am32, fixed, need more test, draft now

File: dronecan_gui_tool/widgets/node_properties.py [hunk 1]
```diff
@@ -362,6 +369,7 @@ def __init__(self, parent, node, target_node_id, param_struct, update_callback):
         self._target_node_id = target_node_id
         self._param_struct = param_struct
         self._update_callback = update_callback
+        self._is_tune_editor = param_struct.name == 'TUNE'
```
Comment (tridge) [inline dronecan_gui_tool/widgets/node_properties.py:372]: maybe we should rename it to STARTUP_TUNE ? and check it is a string?

**Review by tridge — CHANGES_REQUESTED** [review]

File: dronecan_gui_tool/widgets/node_properties.py (hunk 1 above)
Comment (Huibean) [reply dronecan_gui_tool/widgets/node_properties.py:372]: right, can you rename it in AM32?

Comment (Huibean) [reply dronecan_gui_tool/widgets/node_properties.py]: renamed to STARTUP_TUNE, now also can input empty string to restore to the default startup tune

**Review by tridge — APPROVED** [review]

**Pull request merged**